build.gradle

```diff
@@ -165,6 +165,7 @@ def validationXsdPath = project(':sbe-tool').projectDir.toString() + '/src/main/
 project(':sbe-tool') {
     dependencies {
         compile 'org.agrona:agrona:0.9.6'
+        compile 'com.google.guava:guava:21.0'
```

Comment:
We do not have any third party dependencies in in SbeTool other than agrona. Is this required and can it be avoided?